index.spec.js

```diff
@@ -0,0 +1,68 @@
+//import server
+const server = require('./api/server.js');
+
+//import supertest
+const request = require('supertest');
+
+
+describe('GET /games testing',()=>{
+
+  it('should respond with status code 200', async ()=>{
+    const response = await request(server).get('/games');
+    expect(response.status).toBe(200);
+  });
+
+  it('should return type JSON', async ()=>{
+    const response = await request(server).get('/games');
+    expect(response.type).toBe('application/json');
+  });
+
+  it('should return all games, or an empy array if no games', async ()=>{
+    const response = await request(server).get('/games');
+    const expected = (JSON.parse(response.text).length === 0)
+    ? [] 
+    : JSON.parse(response.text); 
+
+    expect(response.body).toEqual(expected);
+  });
+
+});
+
+describe('POST /games testing',()=>{
+
+  it('should respond with status code 201', async ()=>{
+    const response = await request(server).post('/games')
+      .send({
+        "title": "Asteroids",
+        "genre": "Arcade",
+        "releaseYear": "1979"
+      });
+    expect(response.status).toBe(201);
+  });
+
```

Comment:
This block looks good, and you're returning a 422 in your actual endpoint, but I think they want the `status code` to be checked via your tests too. You are returning helpful error messages though so this is _really_ subjective.
`Write tests to verify that the endpoint returns the correct HTTP status code when receiving correct and incorrect game data`
Not a major issue and the test itself would be similar to the test above where you're checking on that 201 status 👍